packages/synapse-react-client/src/lib/umd.index.ts

```diff
@@ -54,19 +54,19 @@ import { SynapseClient, SynapseConstants } from './utils'
 import Palettes from './utils/theme/palette/Palettes'
 import {
   SynapseContextConsumer,
-  SynapseContextProvider,
   useSynapseContext,
 } from './utils/SynapseContext'
 import { SynapseQueries } from './utils'
 import TwoFactorBackupCodes from './containers/auth/TwoFactorBackupCodes'
 import TwoFactorEnrollmentForm from './containers/auth/TwoFactorEnrollmentForm'
 import TwoFactorAuthSettingsPanel from './containers/auth/TwoFactorAuthSettingsPanel'
+import { FullContextProvider } from './utils/FullContextProvider'
 
 // Also include scss in the bundle
 import './style/main.scss'
 
 const SynapseContext = {
-  SynapseContextProvider,
+  SynapseContextProvider: FullContextProvider,
```

Comment:
This file is what we use to create the bundle of SRC components that SynapseWebClient uses. SynapseWebClient needs the FullContextProvider, and maintaining the name prevents this from being a breaking API change.

Having a breaking API change between SRC and SWC is not really a big deal, but this is one less change I have to worry about in a refactor with a lot of moving parts (beyond this PR).